### DISCUSSION TITLE
Adding the man pages after a pre-built binary install on Kubuntu.

### DISCUSSION THREAD

**Discussion by jbb-socuofu** [kovidgoyal/kitty#5374] [General]
I wanted the most recent version of kitty (0.25.2) on my Kubuntu laptop. Instead of the kitty 0.21.2 from the Ubuntu repo. 

After installing the pre-built binaries using the curl installer command. With a Customized install location. I notice I could not pull the man pages up. I saw in the install files there where man pages so I looked up on how to update the man db to use them. I thought that other people might have a "broken" man pages. I am here posting the steps, I did to resolve the problem.
* NOTE: If you customized the install with a different install location. You will need to use the updated path to kitty.app. I suspect that is what caused the issue. I have not tested it yet. 

*# Create man page directories for the kitty man pages
sudo mkdir /usr/local/man/man1
sudo mkdir /usr/local/man/man5

*# Copy kitty man pages
sudo cp ~/.local/kitty.app/share/man/man1/kitty.1 /usr/local/man/man1
sudo cp ~/.local/kitty.app/share/man/man5/kitty.conf.5 /usr.local/man5

*# Compress kitty man pages
sudo gzip /usr/local/man/man1/kitty.1
sudo gzip /usr/local/man/man5/kitty.conf.5

*# Update man database
sudo mandb

**Comment by kovidgoyal** [comment]
Just do

export MANPATH=":$HOME/.local/kitty.app/share/man"

in your shell's rc files.

**Comment by kovidgoyal** [comment]
Or even better:

https://github.com/kovidgoyal/kitty/commit/df3846fbb83eba02de6a0388e911e5615dd825f1

**Reply by jbb-socuofu** [reply]
I was hopping you would see this discussion and add something to the desktop integration info :)
